app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @posts.each do |post|%>
```

Comment:
Kindly remove this logic from your `views/posts/_post.html.erb`.

Reply:
Can you show that you did the same thing on your project?